packages/opentelemetry-resources/src/index.ts

```diff
@@ -16,3 +16,4 @@
 
 export { Resource } from './Resource';
 export * from './constants';
+export * from './resource-assertions';
```

Comment:
I think no need to export internal assertion util,

Reply:
Both this and https://github.com/open-telemetry/opentelemetry-js/pull/846#discussion_r389856450 are so that other packages can use the utilities to validate resources. Currently this is used in tests in opentelemetry-web, opentelemetry-node, opentelemetry-metrics, opentelemetry-tracing. Let me know if this should change.

Reply:
I see, but I am not sure if this is the right approach. Would like to hear from others.

Reply:
It's probably sufficient for me to verify that a resource exists on the proper telemetry objects, without having to assert that it conforms to a specific semantic convention. I can got that route if it'd be better. Just let me know.

Reply:
why not add it to `@opentelemetry/test-utils`?

Reply:
I think this will create a circular dependency that I'm not sure how to resolve. If the resource-assertions move to test-utils, they will have a dependency on opentelemetry-resources. opentelemetry-resources will want to use the resource-assertions, so it will depend on test-utils.